PPAK_CVS/Sources/Storage/Storage.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Storage.swift
+//  PPAK_CVS
+//
+//  Created by 김응철 on 2022/12/25.
+//
+
+import Foundation
+
+final class Storage {
+  
+  static let shared = Storage()
+  
+  private let key = "Storage"
+  private let userDefaults = UserDefaults.standard
+  private init() {}
+  
+  private(set) var products: [ProductModel] = shared.load() {
```

Comment:
클래스 내부 메서드를 사용하시는 게 적절할 것 같아요.
```suggestion
  private(set) lazy var products = load() {
```